### PR TITLE
release announcement 2.2.1

### PR DESCRIPTION
**Pull request by hsbt**
Can anyone review this?

### CODE REVIEW

File: en/news/_posts/2015-03-03-ruby-2-2-1-released.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+author: hsbt"
+translator:
+date: 2015-03-03 03:00:00 +0000
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.2.1.
+This is the first patchlevel release of the stable 2.2 series.
+
+This release include the fix of ffi build failure and memory leak issue on Symbol GC [Bug #10686].
```
Comment (JuanitoFatas) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: Missing `(https://bugs.ruby-lang.org/issues/10686)`.

```
[Bug #10686](https://bugs.ruby-lang.org/issues/10686)
```

:smile:

Comment (JuanitoFatas) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: Put the bug link inside parenthesis? =>

```
This release include the fix of ffi build failure and memory leak issue on Symbol GC (See [Bug #10686](https://bugs.ruby-lang.org/issues/10686)).
```

File: en/news/_posts/2015-03-03-ruby-2-2-1-released.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+layout: news_post
+title: "Ruby 2.2.1 Released"
+author: hsbt"
```
Comment (marocchino) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: > author: "hsbt"

**Comment by JuanitoFatas** [comment]
:+1:

**Comment by marocchino** [comment]
:+1:

**Comment by chikamichi** [comment]
Nice.

File: en/news/_posts/2015-03-03-ruby-2-2-1-released.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+layout: news_post
+title: "Ruby 2.2.1 Released"
+author: "hsbt"
+translator:
+date: 2015-03-03 03:00:00 +0000
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.2.1.
+This is the first patchlevel release of the stable 2.2 series.
```
Comment (amatsuda) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: Do we still call this a "patchlevel release"?

File: en/news/_posts/2015-03-03-ruby-2-2-1-released.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+This release include the fix of ffi build failure and memory leak issue on Symbol GC (See [Bug #10686](https://bugs.ruby-lang.org/issues/10686)).
```
Comment (amatsuda) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: includes

Comment (amatsuda) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: fix for

File: en/news/_posts/2015-03-03-ruby-2-2-1-released.md (hunk 3 above)
Comment (hsbt) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: I replaced "tiny version release"

**Comment by hsbt** [comment]
@marocchino @JuanitoFatas @amatsuda Thanks for your review!

File: en/news/_posts/2015-03-03-ruby-2-2-1-released.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+layout: news_post
+title: "Ruby 2.2.1 Released"
+author: "hsbt"
+translator:
+date: 2015-03-03 03:00:00 +0000
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.2.1.
+This is the first tiny version release of the stable 2.2 series.
```
Comment (JuanitoFatas) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: @hsbt TEENY instead of tiny? :confused:

Comment (hsbt) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: oops :imp:

Comment (JuanitoFatas) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: :+1: :beers:

Comment (marocchino) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: (semver)[http://semver.org/] says this as minor. We use different versioning?

Comment (JuanitoFatas) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: @makimoto Yup. We use a slightly different semantic versioning, please see: https://www.ruby-lang.org/ko/news/2013/12/21/ruby-version-policy-changes-with-2-1-0/.

Comment (marocchino) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: @JuanitoFatas Thank you!

Comment (JuanitoFatas) [inline en/news/_posts/2015-03-03-ruby-2-2-1-released.md]: :wink: Anytime.

**Pull request merged**